src/EditorFeatures/CSharpTest/TextStructureNavigation/TextStructureNavigatorTests.cs

```diff
@@ -254,7 +254,7 @@ public void String()
                 TestString,
                 pos: TestString.LastIndexOf('"'),
                 isSignificant: true,
-                start: startOfString + lengthOfStringIncludingQuotes - 1, length: 2);
+                start: startOfString + lengthOfStringIncludingQuotes - 1, length: 1);
 
             AssertExtent(
                 TestString,
```

Comment:
We already had tests for C# interpolated strings that were correct; only regular/raw strings needed to be updated to treat the closing quote as a single token.